apisix/discovery/eureka/init.lua

```diff
@@ -16,15 +16,14 @@
 --
 
 local local_conf         = require("apisix.core.config_local").local_conf()
-local http               = require("resty.http")
+local socket_http        = require("socket.http")
 local core               = require("apisix.core")
```

Comment:
We don't need to use blocking sockets at every phase, you can take a look at https://github.com/apache/apisix/blob/master/apisix/patch.lua and make some adjustments

Reply:
> We don't need to use blocking sockets at every phase, you can take a look at https://github.com/apache/apisix/blob/master/apisix/patch.lua and make some adjustments

Ok，let me try.